plumbing/format/packfile/delta_selector_test.go

```diff
@@ -197,3 +197,8 @@ func (s *DeltaSelectorSuite) TestObjectsToPack(c *C) {
 	c.Assert(otp[2].IsDelta(), Equals, true)
 	c.Assert(otp[2].Depth, Equals, 2)
 }
+
+func (s *DeltaSelectorSuite) TestMaxDepth(c *C) {
+	dsl := s.ds.deltaSizeLimit(0, 0, 50, true)
```

Comment:
Can we test this on a public function instead? private functions can change (or be removed) at any time, which may remove this test in the future.